streaming_providers/torbox/client.py

```diff
@@ -31,11 +31,16 @@ def _make_request(
             method, url, data, params, is_return_none, is_expected_to_fail
         )
 
-    def add_magent_link(self, magnet_link):
-        response_data = self._make_request("POST", "/torrents/createtorrent", data={"magnet": magnet_link})
+    def add_magnet_link(self, magnet_link):
+        response_data = self._make_request(
+            "POST",
+            "/torrents/createtorrent",
+            data={"magnet": magnet_link},
+            is_expected_to_fail=True
+        )
 
-        # Response when created successfully shows up as "Torrent queued successfully"
-        if "successfully" not in response_data.get("detail"):
+        detail = response_data.get("detail")
+        if type(detail) is bool and detail is False:
```

Comment:
```suggestion
        if response_data.get("detail") is False:
```
This condition only enough to type check and falsy value